docs/src/pages/components/menus/BasicMenu.js

```diff
@@ -3,28 +3,35 @@ import Button from '@material-ui/core/Button';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
 
-export default function SimpleMenu() {
+export default function BasicMenu() {
   const [anchorEl, setAnchorEl] = React.useState(null);
-
+  const open = Boolean(anchorEl);
   const handleClick = (event) => {
     setAnchorEl(event.currentTarget);
   };
-
   const handleClose = () => {
     setAnchorEl(null);
   };
 
   return (
     <div>
-      <Button aria-controls="simple-menu" aria-haspopup="true" onClick={handleClick}>
+      <Button
+        id="basic-button"
+        aria-controls="basic-menu"
+        aria-haspopup="true"
+        aria-expanded={open ? 'true' : undefined}
+        onClick={handleClick}
+      >
         Open Menu
       </Button>
       <Menu
-        id="simple-menu"
+        id="basic-menu"
         anchorEl={anchorEl}
-        keepMounted
-        open={Boolean(anchorEl)}
+        open={open}
         onClose={handleClose}
+        MenuListProps={{
+          'aria-labelledby': 'basic-button',
```

Comment:
Now the label is "open menu" which is a poor label. It would lead to "open menu Menu" announcements. Either we find a better label for the button i.e. the menu or remove this prop.

Reply:
Great catch. I assume that our button not having the best label is specific that our documentation, that it would be better to document the labelled-by. I can't think of a better label for the button that wouldn't lead to announcing "menu menu". I don't really have a preference. I think that we can replace "Open menu" to "Basic menu" or remove the prop.

Reply:
We already use non-generic items so we can do the same thing for the button. e.g. "profile actions" or "profile" etc.

Reply:
OK, sounds great.